src/orm.ts

```diff
@@ -47,4 +47,40 @@ export class ORM<T extends IID> implements IORM {
         const docs = await db.findAsync(query)
         return docs
     }
+
+    static async findOne<T>(query: Partial<T>): Promise<T | null> {
+        const db = await ORM.getDatabase()
+        const doc = await db.findOneAsync(query)
+        return doc
+    }
+
+    static async update<T>(query: Partial<T>, update: Partial<T>, options?: any): Promise<number> {
+        const db = await ORM.getDatabase()
+        const { numAffected } = await db.updateAsync(query, { $set: update }, options)
+        return numAffected
+    }
```

Comment:
The save method both creates and updates one specific object, but indeed it cannot be used to do batch updates.